opennmt/utils/hooks.py

```diff
@@ -182,3 +183,33 @@ def end(self, session):
     tf.logging.info("Evaluation predictions saved to %s", self._output_path)
     if self._post_evaluation_fn is not None:
       self._post_evaluation_fn(self._current_step, self._output_path)
+
+class LoadCheckpointHook(tf.train.SessionRunHook):
```

Comment:
I would like to suggest a more explicit name for this hook:

```suggestion
class LoadWeightsFromCheckpointHook(tf.train.SessionRunHook):
```

so that it's clear that it only loads model weights and nothing else.